src/JustEat.StatsD/Buffered/IStatsDBufferedTransport.cs

```diff
@@ -0,0 +1,9 @@
+using System;
+
+namespace JustEat.StatsD.Buffered
+{
+    internal interface IStatsDBufferedTransport
```

Comment:
Did you mean to make this `internal`? Otherwise the transports only support it when using the concrete type.

Reply:
Making it public

Reply:
If this is going to be public then I suggest not putting it in the `Buffered` namespace for discoversbility, as otherwise it’s the only public type in that whole namespace.